beetsplug/subsonicupdate.py

```diff
@@ -110,48 +97,30 @@ def __format_url(endpoint):
 
         return url + '/rest/{}'.format(endpoint)
 
-    def __get_version(self):
-        url = self.__format_url("ping.view")
-        payload = {
-            'c': 'beets',
-            'f': 'json'
-        }
-        try:
-            response = requests.get(url, params=payload)
-            if response.status_code == 200:
-                json = response.json()
-                version = json['subsonic-response']['version']
-                self._log.info(
-                    u'subsonic version:{0} '.format(version))
-                return tuple(int(s) for s in version.split('.'))
-            else:
-                self._log.error(u'Error: {0}', json)
-                return None
-        except Exception as error:
-            self._log.error(u'Error: {0}'.format(error))
-            return None
-
     def start_scan(self):
         user = config['subsonic']['user'].as_str()
-        url = self.__format_url("startScan.view")
+        auth = config['subsonic']['auth'].as_str()
+        url = self.__format_url("startScan")
+        self._log.info(u'URL is {0}', url)
+        self._log.info(u'auth type is {0}', config['subsonic']['auth'])
```

Comment:
These should probably be `debug` logs, as opposed to `info`? (That will make them invisible by default, and I suspect most users don't need to see them.)